frappe/core/doctype/doctype/doctype.json

```diff
@@ -605,6 +606,12 @@
    "fieldname": "make_attachments_public",
    "fieldtype": "Check",
    "label": "Make Attachments Public by Default"
+  },
+  {
+   "default": "0",
+   "fieldname": "submit_in_background",
+   "fieldtype": "Check",
+   "label": "Submit in background"
```

Comment:
should only be shown when `issubmittable` is checked

the `depends_on` property can be used for this - check other examples in this file

Reply:
also this does not seem to be easily accessible.

would be better to show over here

<img width="1083" alt="Screenshot 2022-10-05 at 3 14 41 PM" src="https://user-images.githubusercontent.com/32034600/194031618-17aee9ec-146d-40f1-8a16-5a95b37d97e0.png">

Reply:
done 👍🏿